lib/btc/BtcRpc.js

```diff
@@ -45,15 +45,67 @@ class BtcRpc {
     return this.asyncCall('cmdlineUnlock', [time]);
   }
 
+  async sendMany({ batch, options }) {
+    for (let tx in batch) {
+      batch[tx] /= 1e8;
+    }
+    const array = ['', batch];
+    if (options) {
+      array.push(options);
+    }
+    return this.asyncCall('sendMany', array);
+  }
+
   async sendToAddress({ address, amount }) {
     return this.asyncCall('sendToAddress', [address, amount / 1e8]);
   }
 
+  async unlockAndSendManyBatched({ batchArray, passphrase, options, time = 10800 }) {
+    function RpcException(message) {
+      const e = new Error(message);
+      e.name = 'failedRequest';
+      e.data = txFailureResults;
+      return e;
+    }
+    if (passphrase === undefined) {
+      passphrase = await passwordPromptAsync('> ');
+    }
+    await this.walletUnlock({ passphrase, time: time });
+    this.emitter.emit('unlocked', time);
+    let txSuccessResults = [];
+    let txFailureResults = [];
+    let failureBool;
+    for (let batch of batchArray) {
+      const emitData = {
+        output: batch
```

Comment:
can we add txid as a key here and set it to empty string? That way we'll no from looking at it that we're going to set that later